<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoviesWGUIClass</class>
 <widget class="QMainWindow" name="MoviesWGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoviesWGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="userButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adminButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Administrator</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeMessage">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Please choose the mode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseAction1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Please choose an action:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Movie</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Movie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowAll">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Movie</string>
    </property>
   </widget>
   <widget class="QListWidget" name="MoviesList">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>521</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="genreLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLabel" name="yearLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Release Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="trailerLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Trailer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendAddMovie">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Movie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="genreInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="trailerInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="movieToRemove">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="removeMovieLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Movie to delete(title):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendDeleteMovie">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendUpdateMovie">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleUpdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genreUpdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearUpdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Release Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trailerUpdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Trailer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="likesUpdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Likes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="movieToUpdate">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>33</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Title of the movie to update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitForUser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchMovie">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Movie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFromWatchList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete from watch list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seeWatchList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>See watch list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openWatchList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open watch list</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseAction1_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Please choose an action:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genreSearch">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendSearchMovies">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>127</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textCout">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="yesAdd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noAdd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitSearch">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit search</string>
    </property>
   </widget>
   <widget class="QLabel" name="removeMovieLabel_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Movie to delete from watchlist(title):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="movieToRemove_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendDeleteMovie2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QTableView" name="watchListTable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showTableButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show table</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MoviesWGUI.qrc"/>
 </resources>
 <connections/>
</ui>
